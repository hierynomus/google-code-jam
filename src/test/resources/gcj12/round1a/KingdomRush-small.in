100
2
0 1
0 2
3
2 2
0 0
4 4
1
1 1
5
0 5
0 1
1 1
4 7
5 6
9
6 9
14 18
3 14
0 0
2 9
7 13
5 7
4 5
7 14
9
10 12
3 9
14 14
7 8
1 12
0 3
2 9
14 16
3 7
7
6 8
0 3
3 5
10 11
8 10
0 12
2 5
8
0 2
5 7
5 11
1 13
2 11
5 11
14 14
2 6
7
0 5
0 7
0 2
0 3
0 1
0 2
0 13
8
11 12
2 2
0 12
2 2
12 15
5 9
5 9
2 2
2
0 0
0 0
7
1 12
11 12
1 3
4 5
0 4
4 6
9 9
9
7 8
12 16
0 2
5 5
9 16
1 14
1 11
8 12
2 12
2
0 3
0 3
10
0 11
0 3
10 18
9 12
16 18
6 14
6 13
3 15
6 8
0 11
6
0 5
0 1
0 11
0 4
0 2
0 6
5
0 2
2 6
0 8
4 7
2 8
2
271 876
634 936
8
0 4
0 15
0 7
0 5
0 1
0 3
0 3
0 2
8
4 8
12 12
5 6
8 13
2 4
2 13
1 12
0 5
8
6 8
3 10
1 5
9 13
12 15
2 14
0 4
7 11
10
3 4
8 10
12 15
0 2
5 18
13 14
11 16
17 19
6 7
1 9
5
330 486
608 1603
27 246
560 829
940 1626
3
0 2
2 2
3 4
4
3 4
0 0
2 7
3 7
10
2001 2001
2001 2001
2001 2001
2001 2001
2001 2001
2001 2001
2001 2001
2001 2001
2001 2001
2001 2001
6
0 3
0 3
0 4
0 1
0 4
0 11
2
0 1
2 3
7
0 13
0 2
0 1
0 5
0 5
0 5
0 7
7
0 0
0 0
0 0
0 0
0 0
0 0
0 0
10
0 9
0 19
0 4
0 7
0 1
0 5
0 4
0 8
0 8
0 7
5
67 168
767 790
62 1470
869 1351
1057 1151
8
0 5
0 3
0 6
0 1
0 7
0 8
0 15
0 6
10
9 17
15 17
2 18
6 12
1 10
3 6
3 12
0 16
0 9
11 12
4
0 2
5 5
1 5
2 2
9
609 1735
523 1073
24 113
296 1636
56 179
473 1413
933 1384
468 571
203 292
3
0 1
3 5
2 4
4
36 401
84 159
103 838
528 1771
7
2 7
9 13
2 10
1 2
1 8
2 11
0 13
1
0 2
3
3 4
0 1
1 4
5
0 3
1 1
6 7
4 5
1 7
6
24 1845
596 683
160 261
1180 1662
765 1402
941 1626
10
0 10
0 7
0 1
0 7
0 7
0 7
0 2
0 8
0 19
0 8
4
2 2
5 6
2 6
0 1
4
1 5
0 6
1 5
0 6
1
0 1
8
2 5
12 13
4 10
1 11
0 6
14 15
3 7
8 9
2
0 3
2 2
10
12 16
11 11
2 9
2 17
0 15
4 7
2 8
12 16
4 12
0 10
2
0 2
2 2
8
1 3
2 8
10 15
6 11
12 14
4 9
7 13
0 5
2
1 2
0 1
10
9 12
0 1
18 19
2 13
7 15
10 14
3 6
8 16
5 11
4 17
4
1 6
0 6
4 6
0 3
8
10 11
1 15
5 6
2 4
0 3
7 8
12 13
9 14
9
14 15
4 10
0 9
14 15
3 15
4 10
2 7
0 5
8 12
2
1 3
0 1
10
0 1
0 5
0 2
0 9
0 10
0 4
0 4
0 8
0 19
0 5
3
3 4
0 3
0 2
2
1 3
0 3
9
376 775
238 903
198 478
216 245
719 1555
349 1314
743 1951
393 1005
13 35
1
0 1
4
0 5
4 4
0 4
2 5
2
2 3
0 1
8
2 2
0 5
1 2
2 9
3 9
5 7
9 15
13 14
1
0 1
3
0 4
3 4
0 1
5
1307 1737
1651 1664
1182 1552
379 650
352 480
7
0 1
0 3
0 7
0 13
0 4
0 4
0 7
4
0 3
3 6
2 4
2 5
1
0 0
6
0 5
0 9
4 6
4 9
0 2
8 10
6
2 4
5 11
0 2
1 2
0 10
8 9
9
2 5
15 17
0 1
12 12
6 12
10 13
6 10
2 8
0 7
5
1 5
6 8
5 9
0 0
1 6
10
1 5
13 18
1 17
0 7
5 7
11 12
1 17
7 13
5 6
13 15
9
11 14
2 14
0 3
4 7
7 10
11 12
14 15
4 5
2 9
10
0 9
0 19
0 3
0 1
0 3
0 8
0 10
0 8
0 6
0 7
9
11 15
5 12
0 1
4 10
9 14
5 15
0 12
5 14
3 5
2
0 1
2 3
7
541 675
147 1187
198 869
649 689
464 1658
59 167
688 975
8
14 15
3 4
1 2
0 6
8 9
12 13
10 11
5 7
9
2 15
0 5
3 6
14 17
11 12
9 16
7 10
8 13
1 4
5
1551 1899
117 146
1108 1458
53 766
199 1737
8
6 12
0 5
12 15
3 4
0 7
0 11
4 10
1 14
8
839 1881
1590 1662
170 462
513 606
421 675
1344 1382
1776 1928
635 679
3
4 4
0 1
0 4
1
0 2
4
0 2
3 6
0 2
3 5
5
0 1
7 7
3 8
2 3
3 6
6
3 10
9 11
0 4
1 2
5 7
6 8
8
0 6
0 15
0 7
0 8
0 2
0 3
0 1
0 6
7
3 4
1 9
7 12
1 4
9 11
6 8
0 6
5
1 6
5 6
1 2
2 4
8 9
7
6 10
1 1
7 13
9 11
5 5
0 7
1 10
4
0 4
0 4
0 0
4 5
1
0 2
4
0 1
4 5
6 7
0 3
5
0 6
1 8
3 8
1 4
0 5
