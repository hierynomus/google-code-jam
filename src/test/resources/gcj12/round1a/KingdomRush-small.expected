Case #1: 3
Case #2: 3
Case #3: Too Bad
Case #4: 6
Case #5: Too Bad
Case #6: Too Bad
Case #7: 13
Case #8: Too Bad
Case #9: 8
Case #10: Too Bad
Case #11: 2
Case #12: 11
Case #13: 17
Case #14: Too Bad
Case #15: Too Bad
Case #16: 7
Case #17: 10
Case #18: Too Bad
Case #19: 9
Case #20: Too Bad
Case #21: 16
Case #22: 20
Case #23: Too Bad
Case #24: Too Bad
Case #25: Too Bad
Case #26: Too Bad
Case #27: 7
Case #28: 4
Case #29: 8
Case #30: 7
Case #31: 12
Case #32: Too Bad
Case #33: 9
Case #34: Too Bad
Case #35: 6
Case #36: Too Bad
Case #37: 6
Case #38: Too Bad
Case #39: Too Bad
Case #40: Too Bad
Case #41: 6
Case #42: 7
Case #43: Too Bad
Case #44: 13
Case #45: 7
Case #46: Too Bad
Case #47: 2
Case #48: 16
Case #49: Too Bad
Case #50: 18
Case #51: Too Bad
Case #52: 16
Case #53: 3
Case #54: 20
Case #55: Too Bad
Case #56: 16
Case #57: Too Bad
Case #58: 4
Case #59: 11
Case #60: 5
Case #61: Too Bad
Case #62: Too Bad
Case #63: 2
Case #64: Too Bad
Case #65: 4
Case #66: 13
Case #67: 2
Case #68: 5
Case #69: Too Bad
Case #70: 8
Case #71: Too Bad
Case #72: 1
Case #73: 10
Case #74: 10
Case #75: 16
Case #76: Too Bad
Case #77: Too Bad
Case #78: Too Bad
Case #79: 11
Case #80: 17
Case #81: 4
Case #82: Too Bad
Case #83: 16
Case #84: 18
Case #85: Too Bad
Case #86: 15
Case #87: Too Bad
Case #88: Too Bad
Case #89: Too Bad
Case #90: 7
Case #91: 9
Case #92: 12
Case #93: 9
Case #94: 12
Case #95: Too Bad
Case #96: Too Bad
Case #97: 6
Case #98: Too Bad
Case #99: 7
Case #100: 9